app/helpers/events_helper.rb

```diff
@@ -1,10 +1,18 @@
 module EventsHelper
   def next_event
-    @next_event ||= Eventbrite::EventFinder.find(ENV.fetch("NEXT_EVENT_ID"))
+    OpenStruct.new(
```

Comment:
Does this change the API? It looks like previously `next_event` would have a `url` and `venue` method, but the `venue` method would return a venue object, not a URL as it does here.

Reply:
thanks for catching this